packages/theme-product/theme-product.js

```diff
@@ -3,7 +3,7 @@
  * @param {*} url
  */
 export function getProductJson(handle) {
-  return fetch('/product/' + handle + '.js').then(function(response) {
+  return fetch('/products/' + handle + '.js').then(function(response) {
```

Comment:
Yep, you're right. This was an oversight on our part. Thank you for your contribution.

A test would be great, if you have the capacity to do so.

Reply:
Cool. I've ran a test on this locally and it works just fine. I however discovered with this repo that if you don't have the `@shopify\theme-products` module installed globally Jest test will fail. 
![jest-fail](https://user-images.githubusercontent.com/6742768/55546036-bf0c0b00-56ce-11e9-8e0e-b54d5fac2e33.png)

Reply:
I feel like this issue may be related to this fix here: https://github.com/Shopify/theme-scripts/commit/9349210711bbb8b56f6af567758c4dba3c539c56.

Is it possible you may have an outdated branch that doesn't contain this fix? Perhaps you could also try removing the `node_modules` folder and reinstalling the project's dependencies.

Reply:
I have had look and pulled the latest branch, it's still happening.

Reply:
Did you run `yarn bootstrap`? This will install all lerna packages at the root.

Reply:
That's most likely the culprit - you'll want to ensure that the `dist` folders are created in each respective package, which `yarn bootstrap` will do.

Reply:
Thanks fellas, I ran the command `yarn bootstrap` and it return with a bit of errors mainly referencing that folders already exist, at the same time `yarn test` is working now too.